ups/daf_butler.table

```diff
@@ -9,6 +9,7 @@ setupRequired(utils)
 setupRequired(daf_persistence)
 setupRequired(afw)
 
+envPrepend(PATH, ${PRODUCT_DIR}/bin)
 envPrepend(LD_LIBRARY_PATH, ${PRODUCT_DIR}/lib)
```

Comment:
whereby I notice that we are setting library search paths when we should not be since this is pure python package.